compiler/cmd/generate-openapischema/generate-openapischema.go

```diff
@@ -11,13 +12,24 @@ import (
 )
 
 func main() {
-	var yamlsPath string
+	var (
+		yamlsPath        string
+		existingCRDsPath string
+		forceUpgrade     string // Used to denote the forced upgrade of a data model.
+	)
 	flag.StringVar(&yamlsPath, "yamls-path", "", "Path to directory containing CRD YAML definitions")
+	flag.StringVar(&existingCRDsPath, "existing-CRDs-Path", "", "Path to directory containing existing CRD YAML definitions")
```

Comment:
can the flag name be `existing-crds-path` ( case insensitive)

Reply:
Common initialisms that allow us to use full capital letters for "CRD" Having is acceptable. Do you think this will cause any issues? Currently, the compiler uses it internally.  @ramramu3433

Reply:
no @PavitraS thought using case insensitive would reduce users confusion , but this may not exposed directly so should be good